### DISCUSSION TITLE
Keyboard shortcut to set display brightness to a specific level (25% for example)

### DISCUSSION THREAD

**Discussion by davidahale** [MonitorControl/MonitorControl#782] [Ideas]
I'm on video conferences all day showing my video. I have two external LG displays. When I'm on video I need to dim the brightness of all the displays (2 LGs and the internal MBP). I use Keyboard Maestro to send a keyboard command to Scenecuts to control a HomeKit scene and set DND. I'd like to be able to also send a command to set the brightness level. At the end of the video conference I have another Keyboard shortcut via Maestro that changes the HomeKit scene back and turns off DND.

**Comment by Muirium** [comment]
This ought to handle it: just look at the example commands!

https://github.com/waydabber/m1ddc

Familiar sounding developer. 😁

I’ve not used keyboard maestro but I assume it can run shell scripts or apple scripts. Karabiner certainly can.
